.github/actions/build_wrappers/src/build_wrappers.ts

```diff
@@ -0,0 +1,117 @@
+import * as core from "@actions/core";
+import * as cache from "@actions/cache";
+import * as folderHash from "folder-hash";
+import * as crypto from "crypto";
+import * as utils from "./utils/common";
+
+
+async function run(): Promise<void>
+{
+    // TODO to make this general purpose, we need to pass this either from a conf file or as input. Same for the next line
+    const paths = [ "./wrappers/build/" ];
```

Comment:
You probably don't want to hash the build folder - that gets generated after you build. Instead, you probably want to just hash the entire `wrappers` folder immediately after checkout.